src/go/rpk/pkg/cli/cluster/config/get.go

```diff
@@ -69,3 +86,34 @@ output, use the 'edit' and 'export' commands respectively.`,
 
 	return cmd
 }
+
+func getCloudConfig(ctx context.Context, cfg *config.Config, p *config.RpkProfile, configName string) (string, error) {
+	cloudClient := publicapi.NewCloudClientSet(cfg.DevOverrides().PublicAPIURL, p.CurrentAuth().AuthToken)
+	req := connect.NewRequest(&controlplanev1.GetClusterRequest{
+		Id: p.CloudCluster.ClusterID,
+	})
+
+	cluster, err := cloudClient.Cluster.GetCluster(ctx, req)
+	if err != nil {
+		var ce *connect.Error
+		if errors.As(err, &ce) {
+			if ce.Code() == connect.CodePermissionDenied {
+				return "", fmt.Errorf("this user does not have permission to get cluster information, please check your role or contact admin")
```

Comment:
[question] How does one checks for roles right now? When you check this in cloud is there a way to set a user permission to 'get cluster information' or how is it worded in cloud?

cc: @kbatuigas to review the error messages.

Reply:
> How does one checks for roles right now?

We check this from public-api, before we make the request to controlplane, we check the role with spicedb, and then fail or process the request to CP
>  When you check this in cloud is there a way to set a user permission to 'get cluster information' or how is it 

worded in cloud?
this is attached to your roles in IAM, example :` https://cloud.redpanda.com/user-accounts/csc076ge6fbj5b0b96og`

I'm not sure if it reponde your questions

Reply:
> We check this from public-api, before we make the request to controlplane, we check the role with spicedb, and then fail or process the request to CP

Ok, but how does the user action that? I meant to ask how we can guide the user to trigger something based on this error.

So it seems like this information is in the Cloud UI so what do you think of

```
return "", fmt.Errorf("this user does not have permission to get cluster information. Check your roles and permissions in the Redpanda Cloud UI")
```

Reply:
Ah no, the user needs to talk with its organization admin in order to provide the correct permissions

Reply:
@andresaristizabal @r-vasquez Is "get cluster information" a specific fine-grained permission available that lets a role/user see the cluster config information? I'm not super clear where it would fall under what we have outlined here: https://docs.redpanda.com/redpanda-cloud/security/authorization/rbac/rbac/#predefined-roles